pkgs/development/python-modules/dask-image/default.nix

```diff
@@ -2,12 +2,12 @@
 , buildPythonPackage
 , fetchPypi
 , dask
-, numpy, toolz # dask[array]
+, numpy
+, toolz # dask[array]
```

Comment:
```suggestion
```
They are both in dask's `propagatedBuildInputs`.

Reply:
Aren't these imports necessary _for_ the `propagatedBuildInputs`? if you remove them you get `error: undefined variable 'numpy'` or similar for `toolz`